src/pages/financial.tsx

```diff
@@ -1,7 +1,101 @@
+import { useState } from 'react';
 import { NextPageWithLayout } from './_app';
+import { Button } from '../components/atoms/Button';
+import Course from '../components/atoms/Course';
+
+//Course Interface
+interface course {
```

Comment:
```suggestion
interface Course {
```

interfaces should be capitalized, same as below